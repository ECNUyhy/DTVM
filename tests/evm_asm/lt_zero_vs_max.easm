// 无符号：0 < 2^256-1 -> 1
PUSH32 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
PUSH1  0x00
LT
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
