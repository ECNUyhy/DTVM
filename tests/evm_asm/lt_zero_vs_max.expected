status: success
error_code: 0
stack: []
memory: '0000000000000000000000000000000000000000000000000000000000000001'
storage: {}
transient_storage: {}
return: '0000000000000000000000000000000000000000000000000000000000000001'
events: []
